pytorch_lightning/trainer/progress.py

```diff
@@ -99,9 +96,7 @@ def from_defaults(cls, **kwargs: Optional[int]) -> 'Progress':
 class LoopProgress:
     """
     Track loop progress during execution.
-
```

Comment:
Are these docstring changes preference or is your linter doing this automatically?

I personally like the spaces

Reply:
Done !